Source/engine/render/text_render.cpp

```diff
@@ -275,6 +275,11 @@ void WordWrapGameString(char *text, size_t width, GameFontTables size, int spaci
 	}
 }
 
+int DrawString(const CelOutputBuffer &out, const char *text, const Point &position, uint16_t flags, int spacing, int lineHeight, bool drawTextCursor)
+{
+	return DrawString(out, text, { position.x, position.y, 0, 0 }, flags, spacing, lineHeight, drawTextCursor);

```

Comment:
I would suggest moving the logic for what to do when w/h == 0 out of DrawString<Rect> and in to DrawString<Point>.

Reply:
I've moved the handling of width 0 out of the DrawString(SDL_Rect) version. This changes the behaviour of that function when called with an SDL_Rect of width 0 so I've updated the comment to document this and replaced the one caller that was using that behaviour with an appropriately sized rectangle.

Changing the handling of height 0 will need another overload introduced, either splitting size into its own argument or having a signature for width only. It'd make sense to use the work done by @julealgon in #2130 and #2131 for this.